libraries/botbuilder-dialogs/scripts/gen-cldr-data.js

```diff
@@ -12,39 +12,27 @@ import * as Spanish from 'cldr-data/main/es/numbers.json';
 */
 
 // Ensure using node 12 because of recursive mkdir
-if (
-    !process.env.GEN_CLDR_DATA_IGNORE_NODE_VERSION &&
-    process.version.split('.')[0] < 'v12'
-) {
+if (!process.env.GEN_CLDR_DATA_IGNORE_NODE_VERSION && process.version.split('.')[0] < 'v12') {
     console.error(`
-Your node version appears to be below v12: ${ process.version }. 
+Your node version appears to be below v12: ${process.version}. 
 This script will not run correctly on earlier versions of node. 
 Set 'GEN_CLDR_DATA_IGNORE_NODE_VERSION' environment variable to truthy to override`);
 }
 
-const fs = require('fs');
-const path = require('path');
-const cp = require('child_process');
-const os = require('os');
+import { copyFileSync, rmdirSync, mkdirSync } from 'fs';
+import { join } from 'path';
+import { spawn } from 'child_process';
+import { tmpdir } from 'os';
```

Comment:
Does this script file has been tested, due to these changes?

Reply:
The script runs correctly